src/server/bootstrap.ts

```diff
@@ -122,48 +126,30 @@ export async function bootstrap() {
     }
 
     // instantiate pool once and pass as context
-    const projectsFiltersThreadPool = Pool(
-      () => spawn(new Worker('../workers/projectsResolverWorker')),
-      options,
-    );
+    const projectsFiltersThreadPool: Pool<ModuleThread<ProjectResolverWorker>> =
+      Pool(
+        () => spawn(new Worker('../workers/projectsResolverWorker')),
+        options,
+      );
 
-    // Create GraphQL server
-    const apolloServer = new ApolloServer({
-      uploads: false,
-      schema,
-      context: async ({ req, res }: any) => {
-        let token: string = '';
-        try {
-          if (!req) {
-            return null;
-          }
+    const apolloServerPlugins = [
+      process.env.DISABLE_APOLLO_PLAYGROUND !== 'true'
+        ? ApolloServerPluginLandingPageGraphQLPlayground({
+            endpoint: '/graphql',
+          })
+        : ApolloServerPluginLandingPageDisabled(),
```

Comment:
This plugin sets the following page:
<img width="1784" alt="image" src="https://user-images.githubusercontent.com/92376054/216228716-3a29347a-26c6-4eca-a29f-cdb0cffba127.png">

Reply:
Compared to simply disabling it like we had it before:
![image](https://user-images.githubusercontent.com/92376054/216228788-ee85242a-eba8-4794-8240-860202935e9d.png)

Reply:
Should we leave it like that or simple return some form of 404 not found like before? 

I don't think it's that important but I think a 404 is more appropriated. A disabled route shouldn't reveal any data.

Reply:
Good point, I don't know how to solve it in a short time. Let's do it in another issue.